src/Continuation.jl

```diff
@@ -427,23 +427,23 @@ function iterate(it::PALCIterable; _verbosity = it.verbosity)
 	# Converge initial guess
 	(verbosity > 0) && printstyled("*********** CONVERGE INITIAL GUESS *************", bold = true, color = :magenta)
 	# we pass additional kwargs to newton so that it is sent to the newton callback
-	u0, fval, isconverged, itnewton = newton(it.F, it.J, it.x0, it.par, newtonOptions; normN = it.normC, callback = it.callbackN, iterationC = 0, p = p0)
+	u0, fval, isconverged, itnewton = newton(it.F, it.J, it.x0, it.par, newtonOptions; normN = it.normC, callback = it.callbackN)
```

Comment:
Differentiating wrt kwargs is not yet supported https://github.com/FluxML/Zygote.jl/issues/446 and leads to this bug https://github.com/FluxML/Zygote.jl/issues/584